### ISSUE TITLE
undefined variable with params

### ISSUE THREAD

**Issue by Dahlgren** [SkaceKamen/sqflint#2]
It would be nice if variables declared using param/params were detected

**Example**
```
params ["_var"];
hint _var;
```
**Current Result**
`Warning: Possibly undefined variable _var at line 2 column 6`

**Comment by SkaceKamen** [comment]
That would be nice indeed. I'll look into it.

**Comment by Dahlgren** [comment]
Thanks! :)

**Comment by SkaceKamen** [comment]
It should work now. I had to update big part of the parser, so there may be breaking changes, so I'll do some testing before creating new release.
